src/components/tools/swot-analysis/steps/SWOTClassifyAlternativeActions/NormalClassifying/SelectClassificationModal.tsx

```diff
@@ -8,12 +8,37 @@ interface SelectClassificationModalProps {
     action: ClassifiedAlternateAction | null
     open: boolean
     classifications: Map<string, Classification>
-    onSelect: (classification: Classification, action: ClassifiedAlternateAction) => void
+    onNoneSelect: (
+        oldClassification: Classification,
+        action: ClassifiedAlternateAction
+    ) => void
+    onSelectOther: (
+        oldClassification: Classification,
+        newClassification: Classification,
+        action: ClassifiedAlternateAction
+    ) => void
+    onSelect: (
```

Comment:
Maybe only use one onSelect callback with oldClassification, new Classification and action as arguments and pass null to oldClassification if the action is firstly selected and null to newClassification if classification is removed. Would make the api a lot cleaner. Request for comments